### DISCUSSION TITLE
How can I get nonlinear capacity drop after prolonged cycling

### DISCUSSION THREAD

**Discussion by Ghardy1024** [pybamm-team/PyBaMM#1415] [Q&A] [answered]
Hey gays, this pack is really nice! But I got some problem while using it.
From issue #836，we know that lithium plating model is from [https://www.sciencedirect.com/science/article/abs/pii/S0378775317307619](url), but how can I get nonlinear capacity drop after prolonged cycling just like the result in that literature?

Really thanks that if someone can help me !!

**Comment by valentinsulzer** [comment, marked as answer]
Good question, it's something we're actively doing research on now to try to figure out. #1398 introduces that exact model so that might do it, but hard to say for sure until we've tried it

**Reply by Ghardy1024** [reply]
Thanks! So,now I just need to wait.Hope you guys can solve this problem

**Reply by Ghardy1024** [reply]
Dear tinosulzer,
I use the code like this:

pb.set_logging_level("NOTICE")
model = pb.lithium_ion.Yang2017()

experiment = pb.Experiment(
    [
        (
            "Charge at 1 C until 4.2 V",
            "Hold at 4.2 V until C/10",
            "Rest for 5 minutes",
            "Discharge at C/10 until 2.8 V",
            "Rest for 5 minutes",
        )
    ]
    * 10
)
sim = pb.Simulation(model, experiment=experiment)
sim.solve(solver=pb.CasadiSolver(mode="safe"))

but it stop at 4th cycle，so how can I get the result like Yang2017 by useing pybamm

**Reply by valentinsulzer** [reply]
Yeah the parameters in our implementation of the Yang model are wrong. I've tried to improve them but it's hard to get it right as they keep some of the parameters secret. Will merge an improvement soon
